inc/compat/themes/compat-theme-cartsy.php

```diff
@@ -36,6 +39,20 @@ public function checkout_layout_hooks() {
 		add_filter( 'fc_enable_checkout_page_template', '__return_false', 10 );
 	}
 
+
+
+	/**
+	 * Very late checkout hooks.
+	 */
+	public function very_late_checkout_hooks() {
```

Comment:
- Minor: Use consistent naming and priority for `late_hooks()` and `very_late_hooks()` functions. If they are the default priority, always use the respective hooks, function names and priority. In case the functions need to run a different priority, use a specific name `..._hooks` for it.

The reason for this is that we will know exactly which hook and priority it is running if the functions is `late_hooks()` or `very_late_hooks()`, using a similar name might get us confused in the future.

See example for `late_hooks()` or `very_late_hooks()` in the main class `inc/checkout-steps.php`.